lib/lita-lean_tc.rb

```diff
@@ -62,6 +66,15 @@ def build_finished(payload)
           create_confirmed
         end
       end
+      # Returns a list of all cards on the Feature request board
+      #
+      def list_feature_request(response)
```

Comment:
This seems to be duplicated logic from the `FeatureRequests` class, can you just call it directly?

For example:

```ruby
msg = FeatureRequests.new(trello_client).to_msg(config.feature_board_id)
response.reply(msg)
```